Code/Framework/AzFramework/AzFramework/Viewport/ClickDetector.cpp

```diff
@@ -13,7 +13,7 @@
 
 namespace AzFramework
 {
-    ClickDetector::ClickDetector()
+    ClickDetector::ClickDetector() : m_detectionState(DetectionState::Nil)
```

Comment:
The [Coding style guide](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#member-initialization) recommends to initialize default values for members in the header file

Reply:
updated